src/Components/Facility/TreatmentSummary.tsx

```diff
@@ -95,7 +95,7 @@ const TreatmentSummary = (props: any) => {
             <h2 className="text-center text-lg">INTERIM TREATMENT SUMMARY</h2>
 
             <div className="text-right font-bold">
-              {date.getDate()}/{date.getMonth()}/{date.getFullYear()}
+              {date.getDate()}/{date.getMonth() + 1}/{date.getFullYear()}
```

Comment:
Import moment and format with its helper. 
```suggestion
moment(date).format("LL"))
```

Reply:
@bodhish Changes done